docs/pages/modules/third-party-library.mdx

```diff
@@ -0,0 +1,335 @@
+
+- [MPAndroidChart by PhilJay](https://github.com/PhilJay/MPAndroidChart)
+- [Charts by @danielgindi](https://github.com/danielgindi/Charts)
+
+The iOS library is inspired by the Android library, so they both have a very similar API and functionality.
+This makes them a good example for this tutorial.
+
+<Step label="1">
+## Creating a new module
+
+First, we need to create a new empty Expo module. We'll create a separate project for this tutorial, but you can also create a new module inside your existing application.
+
+### As a separate project
+
+Alternatively, if you want to create an empty Expo module to be published on npm and used in any Expo application, run:
+
+<Terminal cmd={['$ npx create-expo-module expo-radial-chart']} />
+
+> **Tip**: Since you aren't going to actually ship this library, you can hit <kbd>return</kbd> for all of the prompts to accept the default values.
+
+Next, you can open the newly created `expo-radial-chart` directory and start editing the native code.
+
+### Inside an existing application
+
+If you want to use our new module as a view inside the existing project, you can run this command inside your project directory:
+
+<Terminal cmd={['$ npx create-expo-module --local expo-radial-chart']} />
+
+Next, you can open the newly created `modules/expo-radial-chart` directory and start editing the native code.
+
+</Step>
+
+<Step label="2">
+
+## Run the example project
+
+Now let's run the example project to make sure everything is working. We'll need to start the TypeScript compiler to watch for changes and rebuild the module JavaScript, and separately in another terminal window we'll compile and run the example app.
+
+<Terminal
+  cmdCopy="npm run build"
+  cmd={[
+    '# Run this in the root of the project to start the TypeScript compiler',
+    '$ npm run build',
+  ]}
+/>
+
+<Terminal
+  cmdCopy="cd example && npx expo run:ios"
+  cmd={[
+    '$ cd example',
+    '# Run the example app on iOS',
+    '$ npx expo run:ios',
+    '# Run the example app on Android',
+    '$ npx expo run:android',
+  ]}
+/>
+
+</Step>
+
+<Step label="3">
+## Add native dependencies
+
+Now we can add the native dependencies to the module by editing the `android/build.gradle` and `ios/ExpoRadialChart.podspec` files.
+
+```diff android/build.grade
+dependencies {
+  implementation project(':expo-modules-core')
+  implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:${getKotlinVersion()}"
++ implementation 'com.github.PhilJay:MPAndroidChart:v3.1.0'
+}
+```
+
+```diff ios/ExpoRadialChart.podspec
+    s.static_framework = true
+
+    s.dependency 'ExpoModulesCore'
++   s.dependency 'Charts', '~> 4.1.0'
+
+    # Swift/Objective-C compatibility
+```
+
+</Step>
+
+<Step label="4">
+## Define an API
+
+To use our module in the application without mistakes we'll define the types for our props.
+Our module accepts a list of series, each with a color and a percentage value.
```

Comment:
```suggestion
```

No need to have a separate line and create two sentence fragments. We can use both of them as we're describing "adding props which is the subject here.